.github/workflows/code-qa.yml

```diff
@@ -43,7 +43,24 @@ jobs:
       - name: Run knip checks
         run: npm run knip
 
-  unit-test:
+  test-extension:
+    runs-on: ubuntu-latest
+    env:
+      NODE_OPTIONS: "--max_old_space_size=16384"
```

Comment:
The `ubuntu-latest` machines have 16GB of ram:
- https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories